L1Trigger/L1TGlobal/interface/L1TGlobalUtil.h

```diff
@@ -97,7 +97,12 @@ namespace l1t {
     /// check that the L1TGlobalUtil has been properly initialised
     bool valid() const;
 
-    static void fillDescription(edm::ParameterSetDescription& desc) { L1TGlobalUtilHelper::fillDescription(desc); }
+    static void fillDescription(edm::ParameterSetDescription& desc,
```

Comment:
This new version of `fillDescription` could also be used in `GenericTriggerEventFlag::fillPSetDescription`, where right now the configuration options for `L1TGlobalUtil` are explicitly declared.